app/notifications/validators.py

```diff
@@ -26,8 +26,8 @@
 def check_service_over_api_rate_limit(service, api_key):
     if current_app.config['API_RATE_LIMIT_ENABLED'] and current_app.config['REDIS_ENABLED']:
         cache_key = rate_limit_cache_key(service.id, api_key.key_type)
-        rate_limit = current_app.config['API_KEY_LIMITS'][api_key.key_type]['limit']
-        interval = current_app.config['API_KEY_LIMITS'][api_key.key_type]['interval']
+        rate_limit = service.rate_limit
+        interval = 60
```

Comment:
Should this really be static? It s nice to be able to change it?

Reply:
We decided it'd be better not to change it - it would make it confusing if some services are per minute and other services are per second, for example. What's the need to change it per env/service/etc?

i'm open to us documenting it

Reply:
Shall we deploy this and iterate it if need be?